src/com/google/javascript/jscomp/js/es6/array/copywithin.js

```diff
@@ -30,33 +34,41 @@ $jscomp.polyfill('Array.prototype.copyWithin', function(orig) {
    * @template VALUE
    */
   var polyfill = function(target, start, opt_end) {
+    if (this == null) {
+      throw new TypeError('this is null or not defined');
```

Comment:
This is unnecessary: dereferencing the `length` property will already cause a TypeError if `this` is null or undefined.  The error message is not the clearest, but we want the polyfills to be as small as possible, and I don't think the extra clarity pulls its weight in code size.

Reply:
Removed. Thanks!